Src/CSharpier.Rider/src/main/java/com/intellij/csharpier/ReformatWithCSharpierAction.java

```diff
@@ -27,15 +27,20 @@ public void actionPerformed(@NotNull AnActionEvent e) {
 
     @Override
     public void update(@NotNull AnActionEvent e) {
-        var virtualFile = e.getData(PlatformDataKeys.VIRTUAL_FILE).toString();
+        var virtualFile = e.getData(PlatformDataKeys.VIRTUAL_FILE);
+        if (virtualFile == null) {
+            e.getPresentation().setVisible(false);
+            return;
+        }
+        var virtualFileString = virtualFile.toString();
         var filePrefix = "file://";
```

Comment:
Probably worth extracting this to a constant.
https://stackoverflow.com/a/9639022